packages/rocketchat-ldap/server/ldap.js

```diff
@@ -479,6 +480,16 @@ export default class LDAP {
 
 		try {
 			this.bindSync(dn, password);
+			if (this.options.find_user_after_login) {
+				const searchOptions = {
+					scope: this.options.User_Search_Scope || 'sub'
+				};
+				const result = this.searchAllSync(dn, searchOptions);
```

Comment:
This probably generates a LDAP request using the user dn as login and not the configured LDAP admin account.